DOCUMENTATION.md

```diff
@@ -93,7 +93,9 @@ The repo detail view also provides features for performing the following git ope
 - Fetching changes from remote
 - Adding a new remote repo
 ```
-![repo-card-details](https://user-images.githubusercontent.com/65342122/87232642-1f175400-c3de-11ea-8ead-80cd5ab4c37c.png)
+
+![repo-card-details](https://user-images.githubusercontent.com/65342122/89164057-30deb800-d594-11ea-94d6-d3a330260044.png)
+
 
 ![commit-logs](https://user-images.githubusercontent.com/65342122/88486754-9a136980-cf9d-11ea-94ad-436834cacf07.png)
 
```

Comment:
The commit logs view now has a dynamic loading action button which has not been updated in the snapshot